### PR TITLE
Use jextract to generate bindings

### PR DESCRIPTION
**Pull request by lukellmann**

### CODE REVIEW

File: app/desktop/uwp_helper/build.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+fn main() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+
+    let package_name = env::var("CARGO_PKG_NAME").unwrap();
+    let output_file = target_dir()
+        .join(format!("{}.h", package_name))
+        .display()
+        .to_string();
+
+    let mut config = Config::default();
```
Comment (DRSchlaubi) [inline app/desktop/uwp_helper/build.rs:17]: Why was this changed from the previous syntax?

Comment (lukellmann) [reply app/desktop/uwp_helper/build.rs:17]: i updated cbindgen and there is a new field that isn't public, so the struct can no longer be initialized manually. also `Hstring.h` is needed for jextract to correctly understand the header.

Comment (DRSchlaubi) [reply app/desktop/uwp_helper/build.rs:17]: ok i see
